docs/cellxgene_census_schema.md

```diff
@@ -126,8 +126,9 @@ These assays were selected with the following criteria:
 > * Can be done at the single-cell level.
 > * May include nascent or elongating RNA data.
 > * May be targeted to specific genes in an assay-specific manner.
+> * May include spatial data only from Visium or Slide-seq.
+> * Doesn’t measure spatial data from other assays,
```

Comment:
Editorial.

Editorial. assays, -> assays.

or you could cut line 130 and rewrite line 129 to:

May include spatial data exclusively from Visium or Slide-seq.

130 reads a bit redundant to me.

Reply:
Made this change in 12a467602b47